shoplist/src/main/java/com/netguru/codereview/network/ShopListApi.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.network
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import kotlinx.coroutines.flow.Flow
+
+interface ShopListApi {
```

Comment:
The ShopListApi should be replaced with ShopListDao (with the same methods) and extracted from network package as the DAO does not have to be network related. ShopListApiMock should implement the DAO.